packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -7,9 +7,9 @@ import {View} from "@khanacademy/wonder-blocks-core";
 
 import ModalCloseButton from "./modal-close-button.js";
 import ModalContent from "./modal-content.js";
-import ModalTitleBar from "./modal-title-bar.js";
 import ModalHeader from "./modal-header.js";
 import ModalFooter from "./modal-footer.js";
+import {Toolbar} from "@khanacademy/wonder-blocks-toolbar";
```

Comment:
🛑 nit: I'd move this up to be with Color/View and the other things from other modules.